pkg/controllers/state/suite_test.go

```diff
@@ -1280,6 +1306,104 @@ var _ = Describe("Inflight Nodes", func() {
 			ExpectReconcileSucceeded(ctx, nodeClaimController, client.ObjectKeyFromObject(nodeClaim))
 			ExpectStateNodeNotFoundForNodeClaim(nodeClaim)
 		})
+		It("should ignore node updates if nodes don't have provider id and are owned", func() {
```

Comment:
same question as the previous
```suggestion
		It("should ignore node updates if nodes don't have provider id and are owned for nodeclaims", func() {
```

Reply:
Same comment as above